mifosng-android/src/main/res/values/strings.xml

```diff
@@ -567,6 +578,33 @@
     <string name="failed_to_fetch_center_info">Failed to fetch center info</string>
     <string name="generic_failure_message">Request failed. Please try again.</string>
 
+    <!-- Placeholder strings-->
```

Comment:
No need to create placeholder strings. 
The reason for this is that, during the runtime they will always be replaced or hidden for some reason. So it doesn't make sense to ship them in the APK, they are just increasing the size of the APK and nothing else.